src/ConfigManager.cpp

```diff
@@ -434,16 +434,25 @@ void ConfigManager::createBaseWebServer() {
 void ConfigManager::streamFile(const char* file, const char mime[]) {
   SPIFFS.begin();
 
-  File f = SPIFFS.open(file, "r");
-  if (!f) {
+  File f;
+  if (file[0] == '/') {
+    f = SPIFFS.open(file, "r");
+  } else {
+    size_t len = strlen(file);
+    char path[len+1] = "/";
+    strcat (path, file);
+    f = SPIFFS.open((char*) path, "r");
+  }
```

Comment:
the `delete` was not making anything happy .. thoughts?

Reply:
guh ... oh well

Reply:
looks better than with the delete. The fewer news the better.

Reply:
do a `strcpy` then `strcat`. That should sort it out.

Reply:
but the char array is a stroke of genius.

Reply:
@nrwiersma builds, doesnt HEAPOUT, passes travis, and does what its supposed to ;)